### PR TITLE
rules-page added

### PR DESCRIPTION
**Pull request by palash23jain**
Screenshot 2022-02-21 at 2 45 11 AM

### CODE REVIEW

File: rules/style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.cric-quiz-intro-title{
+    font-size: var(--fontSize4);
+    font-weight: bold;
+    text-align: center;
+}
+
+.button-container{
+    margin: var(--spacing3) 0;
+    display: flex;
+    justify-content: space-around;
```
Comment (pritam-kr) [inline rules/style.css:27]: you can make utilities for these flex properties

File: rules/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <a href="/index.html">Cric-Quiz</a>
+        </div>
+        <input type="text" placeholder="search" class="navbar-search-input">
+        <div class="navbar-items">
+            <a href="/login/index.html" class="btn btn-warning">Login</a>
+        </div>
+    </div>
+    <main class="home-body">
+        <div class="img-wrapper">
+            <img class="home-img" src="/assets/cric-quiz-home.jpg" alt="" srcset="">
```
Comment (pritam-kr) [inline rules/index.html:26]: put alt attribute to some related text

**Review by BelideNikhil — APPROVED** [review]
Good to go!

File: rules/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            <div class="rules-list">
+                <li>There will be 10 MCQ questions</li>
+                <li>Each question will have 1 correct answer</li>
+                <li>Correct answer gives +1 point</li>
+                <li>Wrong answer gives -1 point</li>
+                <li>Correct answers will be revealed in the end</li>
+                <li>At last, your final score will be revealed</li>
+            </div>
+            <div class="button-container">
+                <a href="/quiz-questions/index.html" class="btn btn-rules">Begin</a>
```
Comment (pritam-kr) [inline rules/index.html:39]: the class name can be better in some of element.

File: rules/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<body>
+    <div class="navbar-container">
+        <div class="navbar-title">
+            <a href="/index.html">Cric-Quiz</a>
+        </div>
+        <input type="text" placeholder="search" class="navbar-search-input">
+        <div class="navbar-items">
+            <a href="/login/index.html" class="btn btn-warning">Login</a>
+        </div>
+    </div>
```
Comment (pritam-kr) [inline rules/index.html:23]: use semantic HTML for navigation

**Review by pritam-kr — APPROVED** [review]
I mentioned all points. That can be fix.

File: rules/index.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rules</title>
+    <link rel="stylesheet" href="https://bitui.netlify.app/common-css/main.css">
+    <link rel="stylesheet" href="/common-css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="navbar-container">
```
Comment (SJTGSHIVAM) [inline rules/index.html:15]: nav could be a good element to use here not even this looks not bad. Consider changing this in a later sprint.

**Review by SJTGSHIVAM — APPROVED** [review]
The rest of the things look good I don't see any major issues. On part of UX, I would say you can improve color selection as discussed over the call the contrast ratios look a bit off but still it looks good to users. We are  good to merge this one.

File: rules/index.html (hunk 5 above)
Comment (palash23jain) [reply rules/index.html:15]: sure will take this into account in next release

**Pull request merged**